### ISSUE TITLE
Allegro API 

### ISSUE THREAD

**Issue by Augenstern24** [allegro/allegro-api#7035]

**Comment by Augenstern24** [comment]
According to this previous link: https://developer.allegro.pl/tutorials/getting-started-with-the-rest-api-0A2x4WaX8iB#how-to-register-a-new-application, we have created sandbox accounts and  have completed the steps inside.
But we don't know what to do next?
How can we connect our officially sold accounts?
Thanks!

best regards
Ina

**Comment by Lukasz-Zurek** [comment]
Please give me your sandbox accounts logins, so I will look at them and give you some instructions. 

If you want to test most of our functionalities/resources, you will need an activated, company account.

Another thing is the fact that you won't be able to connect accounts from the production to the sandbox ones.

**Comment by Augenstern24** [comment]
The account is API-Golpus.
When we were interfacing with Allegro's Api, it said that we needed to be the first to create a sandbox account.
Our ultimate goal was to interface to our activated company account.
But we were only given the steps to guide making a sandbox account, as attached “API”.
We need your help, thanks!
![API](https://user-images.githubusercontent.com/132317821/236125905-8a517a38-2782-4f90-a0d9-95091b5dee72.png)

**Comment by Lukasz-Zurek** [comment]
@Augenstern24 We don't have "API-Golpus" account in our sandbox database. Have you started a registration process? If yes, you should receive an e-mail with a confirmation link.

Just to be clear - first of all you should register your company account through this site: https://allegro.pl.allegrosandbox.pl. Then activate it. Please let me know if there will be any problem with the activation process.

If you have fully activated account on Allegro Sandbox (our test environment) - then you will be able to process through the steps which you have just marked above.

**Comment by Lukasz-Zurek** [comment]
@Augenstern24 Ok, I have found your account - obviously you have a different login (API-Golpus is your application name).

Please go to the [path selection page](https://allegro.pl.allegrosandbox.pl/rejestracja/konto-firmowe/wybierz-sciezke) and choose the one related to sales. At the end, you will have to [set up withdrawals](https://allegro.pl.allegrosandbox.pl/rejestracja/konto-firmowe/ustaw-wyplaty) and fill in the data form again and attach the (fictitious) company documents and make an online transfer (via our payment simulator).

Please let me know once you complete these steps or if you have any issues in this process.

**Comment by Lukasz-Zurek** [comment]
@Augenstern24 Yes, I have already activated your account, but I there is a little problem with the company verification so I will forward it to another department. Please be patient. I will be back with the information.

Another thing - it will be better to fill in fictional data as Allegro Sandbox Is only test environment. If the process is finished, you will be able to use Allegro Sandbox API.

**Comment by Augenstern24** [comment]
fictional data:
does this mean I don't need to fill in the same company information and documents as my current real account?

**Comment by Augenstern24** [comment]
But can I also fill in real information?

**Comment by Lukasz-Zurek** [comment]
@Augenstern24 You can. It is up to you. I just wrote that we suggest using fictional data on the Allegro Sandbox in general.

**Comment by Augenstern24** [comment]
Hello, i have sent the document, Do I need to continue setting up the last two steps? As attached.
![set up](https://user-images.githubusercontent.com/132317821/236196250-e4cd46d1-60be-42e6-be52-21d626b51605.png)

**Comment by Augenstern24** [comment]
But this account is only a sandbox test account and we need to list offer as well?

**Comment by Lukasz-Zurek** [comment]
@Augenstern24 If you want to test Allegro functionalities, you will have to use our test environment where you can also list offers, test order management, etc.

But if you want to list real offers on the production's site, on Allegro.pl, you will have to register company account right [here](https://allegro.pl/rejestracja/konto-firmowe/nowe-konto?origin_url=%2F). Then create application.

You can check the differences between live (production) and test REST API in [our guide](https://developer.allegro.pl/tutorials/basic-information-VL6YelvVKTn#test-environment).

**Comment by Augenstern24** [comment]
Hello, thanks for your reply.
So we must test all the steps inside the sandbox before we can officially create an API for our real account?
Or can we just use the real account to interface using the attached URL?
Thanks!
![api](https://user-images.githubusercontent.com/132317821/236722500-dcf9ba9b-f458-4997-ab14-5ad0505ca9e1.png)

**Comment by Lukasz-Zurek** [comment]
@Augenstern24 You can use a production version already, [creating new company account](https://allegro.pl/rejestracja/konto-firmowe/nowe-konto?origin_url=%2F) and [application](https://apps.developer.allegro.pl). 

But obviously the test environment can be useful for the various tests and the fees are not charged there.

**Comment by Augenstern24** [comment]
Thanks, but i have already a company account.
I am trying to connect the oder information of the account we already have through the API.
Why do I need to create a new company account again?

**Comment by Lukasz-Zurek** [comment]
@Augenstern24 f you already have a company account - that's ok. You don't have to register new account.

**Comment by Augenstern24** [comment]
Thanks.
So we can direct  manage the application, activate our real company account by transferring funds?
Is two-factor authentication compulsory?
If we have completed these two steps, what should we do next?
Is the documentation available?

**Comment by Lukasz-Zurek** [comment]
> Thanks.
> So we can direct manage the application, activate our real company account by transferring funds?

Yes. If you have any problem with the activation process, please use [this contact form](https://allegro.pl/pomoc/kontakt?kategoria=61571da6-f6de-4b8f-9ab9-699c8c797cfc).
> Is two-factor authentication compulsory?

Yes.
> If we have completed these two steps, what should we do next?

You should [register an application](https://developer.allegro.pl/tutorials/authentication-and-authorization-m09BlVyo7iY#application-registration). If you have it already, you can already use one of [our authorization methods](https://developer.allegro.pl/tutorials/authentication-and-authorization-m09BlVyo7iY#introduction). Please note that [Client_credentials flow](https://developer.allegro.pl/tutorials/authentication-and-authorization-m09BlVyo7iY#clientcredentials-flow) is without user context. So we suggest using [authorization code flow](https://developer.allegro.pl/tutorials/authentication-and-authorization-m09BlVyo7iY#authorization-code-flow), which is the most popular.
> Is the documentation available?

[Here](https://developer.allegro.pl/documentation) is a documentation. We recommend reading our [tutorial](https://developer.allegro.pl), [FAQ](https://developer.allegro.pl/faq),etc.

**Comment by Augenstern24** [comment]
When I click on this hyperlink register an application, it shows: You cannot register a new application
Why and can you help check that I am registered?
I already have a real account named Augenstern24, which has been activated and is for sale,
You can see these in attachment.
Thanks for your help.
![real account 1](https://github.com/allegro/allegro-api/assets/132317821/5a87e1a3-53d8-4e71-b266-132fcb44bcc1)
![real account](https://github.com/allegro/allegro-api/assets/132317821/3940ba38-b426-4092-96f5-ef4d455b7c3d)
![application](https://github.com/allegro/allegro-api/assets/132317821/bbea3f9f-9444-4e95-9cab-232da5de00aa)

**Comment by Lukasz-Zurek** [comment]
@Augenstern24 Your account is already activated so you should also enable Two-factor authentication, right [here](https://allegro.pl/moje-allegro/moje-konto/dwustopniowe-logowanie).

**Comment by Augenstern24** [comment]
Thanks! 
There is another issue that we find strange.
We haven't actually interfaced with the API successfully.
But now on our now real account, the tracking number is already being uploaded automatically.
Exactly the same tracking number and logistics provider as the one we generate.
Do you perhaps know the reason?

**Comment by Augenstern24** [comment]
The tracking number is 83629527571 by GLS, and the order number is 6a393f30-edba-11ed-9d9a-b17d2340ee62

**Comment by Lukasz-Zurek** [comment]
As a service - we automatically provide shipment numbers and statuses from carriers who integrate their systems with Allegro. [Here](https://allegro.pl/dla-sprzedajacych/przewoznicy-zintegrowani-z-allegro-4RY6gLKLZSD#) you can find the full list. But this topic is not stricte API related. If you use "Ship with Allegro" API resources - this situation is not a bug. But if you have any additional questions about this process as a whole - please use [our contact form](https://allegro.pl/pomoc/kontakt?kategoria=44362df6-b15a-4eac-a24c-c534495b1bda).

**Comment by Augenstern24** [comment]
ok, understand.
Then there are no more problems about this.
I will continue to interface with the API.
Thanks!

**Comment by Augenstern24** [comment]
hello, how can i get the key?
Where in the article can you tell me where to operate to get the account key to interface with allegro?
Thanks!

**Comment by Lukasz-Zurek** [comment]
@Augenstern24 What key do you mean? Do yo mean _ClientId_ (your application ID) from the https://apps.developer.allegro.pl site? I recommend reading [our guide](https://developer.allegro.pl/tutorials/jak-zaczac-prace-z-rest-api-4RAvV2bgwtj).

**Comment by Augenstern24** [comment]
Thanks, so i should Allegro application in https://apps.developer.allegro.pl/new registered? 
Can you please tell me how to fill in and choose?
And I'm not sure what I should fill in Application path?
![api](https://github.com/allegro/allegro-api/assets/132317821/86026d6d-646a-453e-bafd-61c183f7f2ca)

**Comment by Lukasz-Zurek** [comment]
@Augenstern24 Please read [our guide](https://developer.allegro.pl/tutorials/uwierzytelnianie-i-autoryzacja-zlq9e75GdIR#rejestracja-aplikacji). We described the whole process.

Please note that the application type is up to you and your needs. And on your print screen you can see that every field is described.

**Application name** is displayed to users when they grant your app consent to access their account. First checkbox in the **Select application type** refers to [Authorization code flow](https://developer.allegro.pl/tutorials/authentication-and-authorization-m09BlVyo7iY#authorization-code-flow) where you should give application path - where the users will be redirected to it once they have linked the account with your app and second checkbox refers to [Device code flow](https://developer.allegro.pl/tutorials/authentication-and-authorization-m09BlVyo7iY#device-flow), where there is no need to add application path.

**Comment by Lukasz-Zurek** [comment]
@Augenstern24 I can confirm that your account Augenstern24 is already active and has two-step verification enabled.

To be able to register an application to use our API, the Allegro account is needed, so in general, it does only correspond to this. But obviously you can link the app to this account through authorization process.
